packages/storybook-vue/stories/3_components/badge/Badge.stories.mdx

```diff
@@ -0,0 +1,289 @@
+import {
+  Meta,
+  Story,
+  ArgsTable,
+  Canvas,
+  Description,
+} from '@storybook/addon-docs/blocks';
+import ScaleBadge from './ScaleBadge.vue';
+
+<Meta
+  title="Beta Components/Badge"
+  component={ScaleBadge}
+  argTypes={{
+    color: {
+      defaultValue: 'magenta',
+      options: ['magenta', 'white', 'black', 'blue'],
+      control: {
+        type: 'select',
+      },
+    },
+    size: {
+      defaultValue: 'big',
+      options: ['small', 'big'],
+      control: {
+        type: 'select',
+      },
+    },
+  }}
+/>
+
+export const Template = (args, { argTypes }) => ({
+  components: { ScaleBadge },
+  props: {
```

Comment:
let's just make this `props: ScaleBadge.props` - the rest is not needed. Same goes for the other templates

Reply:
The same goes for all the other occurrences in this file. thanks!